react/features/invite/functions.ts

```diff
@@ -492,7 +492,7 @@ export function isAddPeopleEnabled(state: IReduxState): boolean {
 export function isDialOutEnabled(state: IReduxState): boolean {
     const { conference } = state['features/base/conference'];
 
-    return isLocalParticipantModerator(state)
+    return (isLocalParticipantModerator(state) || isJwtFeatureEnabled(state, 'outbound-call', false, false))
```

Comment:
This always returns `true` when the user is a moderator. Shouldn't it do something like this (the last param is `ifNotInFeatures`)?
```
return isJwtFeatureEnabled(state, 'outbound-call', false, isLocalParticipantModerator(state))
```